tests/e2e/helpers/loadExtension.ts

```diff
@@ -170,48 +170,77 @@ export async function loginToExistingAlbyAccount(page: Page) {
         timeout: 20000,
       }
     );
-  console.info("Found target: " + newTarget.url());
+  // console.info("Found target: " + newTarget.url());
   //get the new page object:
   const oauthPage = await newTarget.page();
   if (!oauthPage) {
     throw new Error("OAuth page not found");
   }
   const oauthDocument = await getDocument(oauthPage);
 
-  const oauthLoginButton = await getByText(oauthDocument, "Log in to connect");
-  oauthLoginButton.click();
+  const connectButtonText = process.env.CI
```

Comment:
Can we maybe just make the search less specific?

Reply:
I tried but wasn't successful, seems it needs a specific selector if you search for a substring. Regex search also didn't work for me. This shouldn't change now, I would just keep it as is if that's ok